lib/usdUfe/ufe/UsdSceneItem.cpp

```diff
@@ -15,11 +15,25 @@
 //
 #include "UsdSceneItem.h"
 
+#include "Utils.h"
+
 #include <pxr/base/tf/type.h>
 #include <pxr/pxr.h>
 #include <pxr/usd/usd/primTypeInfo.h>
 #include <pxr/usd/usd/schemaRegistry.h>
 
+#ifdef UFE_SCENEITEM_HAS_METADATA
+
+#include "UsdUndoableClearSceneItemMetadataCommand.h"
+#include "UsdUndoableSetSceneItemMetadataCommand.h"
```

Comment:
According to our coding standard, only the corresponding header file should be include this way. All others should use the form:
#include <usdUfe/ufe/xxx.h>